flutter/lib/backend/backends_list.in

```diff
@@ -1,5 +1,6 @@
 part of 'bridge.dart';
 
+// To add a new backend, add an entry for your backend here. For instance, for Intel add "INTEL_TAG"

```

Comment:
Please, add the information that the order matters.
Currently there is no info that only the first matching backend will be used in the docs.
Well, at least I forgot to add it to the Flutter docs, not sure if Android app mentions it.

Reply:
Will do